community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
```

Comment:
Ok, in Debian:

```sh
apt-cache show postgresql-9.5 2> /dev/null
```

This prints stuff about the available package to stdout. Can capture to a variable and test for empty string, or use exit code.